### PR TITLE
HER- 42 replace use navigate with link

### PR DESCRIPTION
**Pull request by mhope21**
HER-42: Replace useNavigate with Link [FE]
## Issue

https://raquelanaroman.atlassian.net/browse/HER-42

Description

The FE ui current makes heavy use of the React Router useNavigate hook.

Per the [documentation](https://reactrouter.com/start/library/navigating#usenavigate):

For normal navigation, it's best to use Link or NavLink. They provide a better default user experience like keyboard events, accessibility labeling, "open in new window", right click context menus, etc.

Reserve usage of useNavigate to situations where the user is not interacting but you need to navigate, for example:

After a form submission completes

Logging them out after inactivity

Timed UIs like quizzes, etc.

 

This is particularly undesirable because it does not easily pass state between pages, but Link allows us to via the [LinkProps.state attribute it accepts as a second arg.](https://github.com/remix-run/react-router/blob/74737750ca0abab137859d445e1a66eb59a56ba2/packages/react-router/lib/dom/lib.tsx#L558-L573)

## Changes

* Use Link instead of useNavigate() in Kits component

### Review Checklist

- [x] I have documented my code with code comments.

### CODE REVIEW

File: frontend/src/components/pages/Contact.jsx [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import React, { useState, useEffect } from 'react'
-import { useNavigate } from 'react-router-dom';
+import { Navigate, useNavigate } from 'react-router-dom';
```
Comment (raquii) [inline frontend/src/components/pages/Contact.jsx:2]: Remove unused import
```suggestion
import { useNavigate } from 'react-router-dom';
```

File: frontend/src/components/pages/Contact.jsx [hunk 2]
```diff
@@ -46,15 +47,9 @@ function Contact({ user }) {
...
-                // Added code to clear first and last name
-                console.log(response.json)
-                setName("");
-                setEmail("");
-                setPhone("");
-                setMessage("");
-                setErrorMessages("");
                 alert("Contact submitted successfully!");
+                navigate("/");
+
```
Comment (raquii) [inline frontend/src/components/pages/Contact.jsx:52]: remove trailing whitespace

```suggestion
```

**Review by raquii — APPROVED** [review]
Comments are non-blocking

**Review by bbcc33 — APPROVED** [review]
Looks good to me!

**Review by trca831 — APPROVED** [review]
LGTM!

**Pull request merged**